### PR TITLE
files added

### PR DESCRIPTION
**Pull request by makivka**

### CODE REVIEW

**Comment by makivka** [comment]
Тут можуть писати коментарі всі, хто хоче?

**Review by ParadoxMH — CHANGES_REQUESTED** [review]

File: ComCompany/Car.cs [hunk 1]
```diff
@@ -0,0 +1,53 @@
+﻿using System.Text;
+
+namespace Company
+{
+    class Car
+    {
+        public string CarModel { get; set; }
+        public string CarClass { get; set; }
```
[the remark this answers is not in the document]
Comment (makivka) [reply ComCompany/Car.cs:8]: +

File: ComCompany/Driver.cs [hunk 2]
```diff
@@ -0,0 +1,12 @@
+﻿namespace Company
+{
+    class Driver : Person
+    {
+        //public string Name { get; set; }
```
[the remark this answers is not in the document]
Comment (makivka) [reply ComCompany/Driver.cs:5]: +

File: ComCompany/Car.cs [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+            turnRight.Append("поворот направо.");
+            return turnRight;
+        }
+        public StringBuilder TurnLeft()
+        {
+            StringBuilder turnLeft = new StringBuilder();
+            turnLeft.Append(CarModel + ", ");
+            turnLeft.Append("поворот налiво.");
+            return turnLeft;
+        }
```
[the remark this answers is not in the document]
Comment (makivka) [reply ComCompany/Car.cs:39]: +

File: ComCompany/Car.cs [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+        public virtual StringBuilder CarInformationToString()
```
[the remark this answers is not in the document]
Comment (makivka) [reply ComCompany/Car.cs:40]: можна поподробней?
я override цей метод для Lorry i SportCar

[the remark this answers is not in the document]
Comment (ParadoxMH) [reply ComCompany/Car.cs:39]: Read about Object class. There is a `ToString()` method, which can be overridden in any class.

You could override the  `ToString()` method in Lorry and SportCar classes too

[the remark this answers is not in the document]
Comment (makivka) [reply ComCompany/Car.cs:39]: done
🤯

File: ComCompany/Car.cs [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+        {
+            StringBuilder move = new StringBuilder();
+            move.Append(Model + ", ");
+            move.Append(direction);
+            return move;
+        }
+        public StringBuilder Start()
+        {
+            StringBuilder start = new StringBuilder();
+            return start=Move("поїхали!");
```
Comment (ParadoxMH) [inline ComCompany/Car.cs:22]: can't you just `return Move("поїхали!")`?

File: ComCompany/Car.cs (hunk 4 above)
[the remark this answers is not in the document]
Comment (ParadoxMH) [reply ComCompany/Car.cs:39]: do you still need `CarInformationToString()` method?

File: ComCompany/Lorry.cs [hunk 6]
```diff
@@ -0,0 +1,34 @@
+﻿using System.Text;
+
+namespace Company
+{
+    class Lorry : Car
+    {
+        public double CarryingCapacity { get; set; }
+        public override StringBuilder CarInformationToString()
```
Comment (ParadoxMH) [inline ComCompany/Lorry.cs:8]: do you still need CarInformationToString() method?

File: ComCompany/Program.cs [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+                Driver = janineDriver,
+                Engine = gasoline_1_5,
+                MaxSpeed = 170
+            };
+
+            Console.WriteLine(jeepRenegade.Start());
+            Console.WriteLine(jeepRenegade.TurnRight());
+            Console.WriteLine(jeepRenegade.Stop());
+            Console.WriteLine(jeepRenegade.TurnLeft());
+            Console.WriteLine(jeepRenegade.ToString());
```
Comment (ParadoxMH) [inline ComCompany/Program.cs:70]: try this - `Console.WriteLine(jeepRenegade);`

File: ComCompany/SportCar.cs [hunk 8]
```diff
@@ -0,0 +1,34 @@
+﻿using System.Text;
+
+namespace Company
+{
+    class SportCar : Car
+    {
+        public double MaxSpeed { get; set; }
+        public override StringBuilder CarInformationToString()
```
Comment (ParadoxMH) [inline ComCompany/SportCar.cs:8]: do you still need CarInformationToString() method?

**Review by ParadoxMH — CHANGES_REQUESTED** [review]

File: ComCompany/Car.cs (hunk 4 above)
[the remark this answers is not in the document]
Comment (makivka) [reply ComCompany/Car.cs:39]: no, I don`t
but  it's a pity to remove

File: ComCompany/SportCar.cs (hunk 8 above)
Comment (makivka) [reply ComCompany/SportCar.cs]: done

File: ComCompany/Lorry.cs (hunk 6 above)
Comment (makivka) [reply ComCompany/Lorry.cs]: done

File: ComCompany/Car.cs (hunk 5 above)
Comment (makivka) [reply ComCompany/Car.cs]: done

File: ComCompany/Program.cs (hunk 7 above)
Comment (makivka) [reply ComCompany/Program.cs:70]: it works, but I use methods in different order

Comment (ParadoxMH) [reply ComCompany/Program.cs:70]: Exactly, so there is no need to call `ToString()` explicitly

**Review by ParadoxMH — APPROVED** [review]

**Pull request merged**